documentation/query.adoc

```diff
@@ -35,7 +35,7 @@ button::Go back to Query[role=NX_TAB_NAV,tab=query]
 == Get to know the database
 
 A good place to start is to take stock of what you have in your database.
-You access _Database Information_ from the very top of the sidebar.
+You access _Database Information_ from the very top left of the sidebar.
```

Comment:
The sidebar doesn't have anything lined up horizontally, only vertically, so I'm not sure if we need to specify _left_ here.